build-configs/BuildConfigType.js

```diff
@@ -3,61 +3,70 @@
 import type { ThemeType } from './ThemeType'
 import type { TranslationsType } from 'translations'
 
+/**
```

Comment:
```suggestion
/*
```

As this is not a jsdoc comment